src/components/transactions/Withdraw/WithdrawModalContent.tsx

```diff
@@ -183,6 +183,13 @@ export const WithdrawModalContent = ({
         isMaxSelected={isMaxSelected}
         disabled={withdrawTxState.loading}
         maxValue={maxAmountToWithdraw.toString(10)}
+        balanceText={
+          unborrowedLiquidity < underlyingBalance ? (
+            <Trans>Available</Trans>
+          ) : (
+            <Trans>Supply balance</Trans>
+          )
+        }
```

Comment:
@defispartan Updated to handle the edge case

Reply:
Since they're BigNumbers, comparison needs to be `unborrowedLiquidity.lt(underlyingBalance)`